Removes erp meme genitals

BillyBangles opened the pull request:
why the fuck did we even have shit like the vagina dentata what the fuck
if we're going to get more realistic, we might as well have realistic genital options.

RobotAlice commented:
Going to make new PR with croptops and fix the science poncho so Travis doesn't bitch about this.

ursamedium commented:
Travis will bitch, he hasn't been set up properly, don't worry about it too much.

Dinnnertime committed:
removes erp meme genitals

why the fuck did we even have shit like the vagina dentata what the fuck

Dinnnertime committed:
removes sprites

because we don't want this shit back.
also re-adds and renames barbed dicks because i don't want to get
lynched by the furries.
